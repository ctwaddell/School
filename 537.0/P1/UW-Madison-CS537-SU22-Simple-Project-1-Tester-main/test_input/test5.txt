     123qwe
56fwa
123qwe456ads
   
123,./qwe555


dwa651e
sad1w561
ngf651td3
123qw////////////////////////e
