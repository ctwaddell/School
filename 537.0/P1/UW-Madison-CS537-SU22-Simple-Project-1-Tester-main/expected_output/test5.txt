     123qwe
123qwe456ads
123,./qwe555
123qw////////////////////////e
